apps/condo/domains/billing/schema/BillingReceipt.js

```diff
@@ -353,4 +353,5 @@ const BillingReceipt = new GQLListSchema('BillingReceipt', {
 
 module.exports = {
     BillingReceipt,
+    DEFAULT_CATEGORY,
```

Comment:
It's better to move this var to external constant, because you have execute all file.

Reply:
We already have such constant, will replace it